### ISSUE TITLE
Lost last byte .buffer readUntil = 'eof'

### ISSUE THREAD

**Issue by levide** [keichi/binary-parser#45]
line 440 of binary_parser.js
```
Parser.prototype.generateBuffer = function(ctx) {
    if (this.options.readUntil === 'eof') {
        ctx.pushCode('{0} = buffer.slice(offset, buffer.length - 1);',
```
maybe need  buffer.slice(offset, **buffer.length**);',

**Comment by kertal** [comment]
there's now a pull request fixing this issue 
https://github.com/keichi/binary-parser/pull/47

**Comment by keichi** [comment]
PR #47 has been merged and included in release 1.1.6.
